v8go.cc

```diff
@@ -1290,6 +1290,27 @@ void ObjectSet(ValuePtr ptr, const char* key, ValuePtr prop_val) {
   obj->Set(local_ctx, key_val, prop_val->ptr.Get(iso)).Check();
 }
 
+RtnStrings ObjectGetPropertyNames(ValuePtr ptr) {
+  LOCAL_OBJECT(ptr);
+  RtnStrings rtn = {};
+
+  MaybeLocal<Array> maybe_names = obj->GetPropertyNames(local_ctx);
+  Local<Array> names = maybe_names.ToLocalChecked();
+
+  uint32_t length = names->Length();
+  const char** strings = new const char*[length];
```

Comment:
This is being deallocated from Go code using C.free which is resulting in an address sanitizer error (`AddressSanitizer: alloc-dealloc-mismatch (operator new [] vs free)`) because it is allocated with the C++ new operator (`new const char*[length]`) then deallocated with C's `free` function.  Instead, this could be allocated using `malloc` to avoid this problem

```suggestion
  const char** strings = malloc(length * sizeof(const char*));
  if (strings == NULL) {
    // TODO: Handle out of memory errors (https://github.com/rogchap/v8go/issues/273)
  }
```

It looks like

Reply:
Fixed here https://github.com/rogchap/v8go/pull/260/commits/84ff923037accc8f91927c3fc81444845007bdb4